src/components/head.tsx

```diff
@@ -2,12 +2,20 @@ type MetaData = {
   title: string;
   description: string;
   icon: string;
+  ogImage?: string;
+  ogUrl?: string;
+  twitterCard?: string;
+  twitterSite?: string;
 };
 
 const DEFAULT_META_DATA: MetaData = {
   title: "React Tokyo",
   description: "A React community in Tokyo",
   icon: "/images/favicon.png",
+  ogImage: "/images/react-tokyo-logo-opg.png",
```

Comment:
[MUST] 画像のURLは絶対パスじゃないとダメっぽいです！

```suggestion
  ogImage: "https://react-tokyo.vercel.app//images/react-tokyo-logo-opg.png",
```

Reply:
ご指摘ありがとうございます！
説明文と画像URLは絶対パスに修正しました。
お手数ですが、もう再度ご確認お願い致します！

Reply:
ありがとうございます！

説明文いい感じに出てます！

<img width="533" alt="image" src="https://github.com/user-attachments/assets/adb24bc9-07b5-4fcd-b536-349b76816201" />

Reply:
良かったです！早速確認ありがとうございます！！